wire/msgmixpairreq.go

```diff
@@ -209,6 +212,14 @@ func (msg *MsgMixPairReq) BtcDecode(r io.Reader, pver uint32) error {
 		return messageError(op, ErrInvalidMsg, msg)
 	}
 
+	var flags, pairingFlags byte
+	err = readElements(r, &flags, &pairingFlags)
```

Comment:
I'd have to look at a benchmark to be sure, but I'm fairly certain this will cause additional allocs.  It should read directly into `&msg.Flags` and `&msg.PairingFlags` to avoid that like all the other messages.